### ISSUE TITLE
cannot run objectnav_replay.py

### ISSUE THREAD

**Issue by yimengli46** [Ram81/habitat-imitation-baselines#6]
Hi Ram,

I was trying to run `objectnav_replay.py` with the following command.
`python examples/objectnav_replay.py --path data/datasets/objectnav/objectnav_mp3d_70k/train/content/17DRP5sb8fy.json.gz`

And I get the following error. I have never seen this error before.
```
2022-07-20 20:42:34,387 Initializing task ObjectNav-v1
2022-07-20 20:42:34,387 max object cat: 27
2022-07-20 20:42:34,387 cats: dict_values([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27])
Traceback (most recent call last):
  File "examples/objectnav_replay.py", line 93, in <module>
    main()
  File "examples/objectnav_replay.py", line 88, in main
    output_prefix=args.output_prefix
  File "examples/objectnav_replay.py", line 18, in run_reference_replay
    with habitat.Env(cfg) as env:
  File "/home/yimeng/Datasets/habitat-web-baselines/habitat/core/env.py", line 117, in __init__
    dataset=self._dataset,
  File "/home/yimeng/Datasets/habitat-web-baselines/habitat/tasks/registration.py", line 22, in make_task
    return _task(**kwargs)
  File "/home/yimeng/Datasets/habitat-web-baselines/habitat/tasks/nav/object_nav_task.py", line 277, in __init__
    super().__init__(**kwargs)
  File "/home/yimeng/Datasets/habitat-web-baselines/habitat/tasks/nav/nav.py", line 1153, in __init__
    super().__init__(config=config, sim=sim, dataset=dataset)
  File "/home/yimeng/Datasets/habitat-web-baselines/habitat/core/embodied_task.py", line 249, in __init__
    entities_config=config,
  File "/home/yimeng/Datasets/habitat-web-baselines/habitat/core/embodied_task.py", line 277, in _init_entities
    task=self,
  File "/home/yimeng/Datasets/habitat-web-baselines/habitat/tasks/pickplace/pickplace.py", line 220, in __init__
    self.observation_space = spaces.Discrete(0)
  File "/home/yimeng/miniconda3/envs/habitat_web/lib/python3.6/site-packages/gym/spaces/discrete.py", line 37, in __init__
    assert n > 0, "n (counts) have to be positive"
AssertionError: n (counts) have to be positive
I0720 20:42:34.562124 1109316 PhysicsManager.cpp:33] Deconstructing PhysicsManager
I0720 20:42:34.562153 1109316 SemanticScene.h:41] Deconstructing SemanticScene
```

**Comment by Ram81** [comment]
Hi @yimengli46, the issue is related to the version of gym API not supporting `0` in observation space. I have pushed a fix in [this commit](https://github.com/Ram81/habitat-imitation-baselines/commit/a7f39b45c050fa953ac898a05b76ddd255493153) . Let me know if you are still facing this issue.

**Comment by yimengli46** [comment]
Hi @Ram81, thanks for the quick response.

I also have a comment on a closed issue #3 . 
I don't have the authority to reopen that issue.

Can you take a look at that one as well?

**Comment by yimengli46** [comment]
Works like a charm.

Thank you @Ram81.
